lib/MetaCPAN/Document/Release.pm

```diff
@@ -591,5 +592,77 @@ sub requires {
     };
 }
 
+sub _activity_filters {
+    my ( $self, $params, $start ) = @_;
+    my ( $author, $distribution, $module, $f )
```

Comment:
this is from the original code in WEB, dating back to 2012, i'll rename to `$new_dists`